content/docs/extensions/pg-extensions.md

```diff
@@ -5,84 +5,85 @@ redirectFrom:
-| tablefunc                | [1.0](https://www.postgresql.org/docs/14/tablefunc.html)        | [1.0](https://www.postgresql.org/docs/15/tablefunc.html)         |                                                                                                                    |
-| tcn                      | [1.0](https://www.postgresql.org/docs/14/tcn.html)              | [1.0](https://www.postgresql.org/docs/15/tcn.html)               |                                                                                                                    |
-| timescaledb              | [2.10.1](https://docs.timescale.com/about/latest/timescaledb-editions/)              | [2.10.1](https://docs.timescale.com/about/latest/timescaledb-editions/)               |  Only Apache-2 licensed features are supported. Compression is not supported.                                                                                                                 |
-| tsm_system_rows          | [1.0](https://www.postgresql.org/docs/14/tsm-system-rows.html)  | [1.0](https://www.postgresql.org/docs/15/tsm-system-rows.html)   |                                                                                                                    |
-| tsm_system_time          | [1.0](https://www.postgresql.org/docs/14/tsm-system-time.html)  | [1.0](https://www.postgresql.org/docs/15/tsm-system-time.html)   |                                                                                                                    |
-| unaccent                 | [1.1](https://www.postgresql.org/docs/14/unaccent.html)         | [1.1](https://www.postgresql.org/docs/15/unaccent.html)          |                                                                                                                    |
-| unit                     | [7.7](https://github.com/df7cb/postgresql-unit)                 | [7.7](https://github.com/df7cb/postgresql-unit)                  |                                                                                                                    |
-| uuid-ossp                | [1.1](https://www.postgresql.org/docs/14/uuid-ossp.html)        | [1.1](https://www.postgresql.org/docs/15/uuid-ossp.html)         | Double-quote the extension name when installing: `CREATE EXTENSION "uuid-ossp"`                                    |
-| xml2                     | [1.1](https://www.postgresql.org/docs/current/xml2.html)        | [1.1](https://www.postgresql.org/docs/current/xml2.html)         |                                                                                                                    |
+| Extension                | Version (Postgres 14)                               | Version (Postgres 15)  | Version (Postgres 16)                              |Notes                                                                                                                |
```

Comment:
Some of the extensions don't support Postgres 16 yet:
- `pg_jsonschema`
- `pg_graphql`
- `pg_tiktoken`
- `pgx_ulid`

Reply:
Also `timescaledb` doesn't support Postgres 16 yet

Reply:
And we don't install `pg_embedding` to Postgres 16

Reply:
Thank you. Updated.